statics/js/components/graph-layout.js

```diff
@@ -1482,29 +1481,37 @@ TopologyGraphLayout.prototype = {
   updateLinkLabelData: function() {
     var self = this;
 
-    const driver = self.linkLabelFactory();
+    const l2 = self.linkL2LabelFactory();
 
     for (var i in this.links) {
       var link = this.links[i];
 
       if (!link.source.visible || !link.target.visible)
         continue;
-      if (link.metadata.RelationType !== "layer2")
-        continue;
 
-      driver.updateData(link);
+      if (link.metadata.RelationType === "layer2") {
+	l2.updateData(link);
```

Comment:
indentation should be 2 (here its' only 1)
make sure to you space (not tab) throughout

Reply:
FIXED